File: defanging-an-ip-address-1108/solution.js
/**
 * LeetCode problem # 1108, 'Defanging an IP Address'
 * https://leetcode.com/problems/defanging-an-ip-address/
 */

/**
 * @param {string} address
 * @return {string}
 */
var defangIPaddr = function (address) {
  return address.replace(/[.]/g, "[.]");
};

File: defanging-an-ip-address-1108/Solution.java
/**
 * LeetCode problem # 1108, 'Defanging an IP Address'
 * https://leetcode.com/problems/defanging-an-ip-address/
 */

class Solution {
    public String defangIPaddr(String address) {
        return address.replaceAll("\\.", "[.]");
    }
}
